server/src/organizations/EINCheck.ts

```diff
@@ -0,0 +1,5 @@
+export class EINCheck {
+  verified: boolean;
+  einExists: boolean;
+  actualName: string;
+}
```

Comment:
this is only being used as a type so let's just make it a type
`type EINCheck = {...`
also, it's only used in the org service, so i'd say we can just keep it local there

Reply:
Copy that. Feeling a little dumb on this one! I think I'd looked in my notes and forgotten that you can make a type w/o it being a class.